### ISSUE TITLE
Unable to attach any server

### ISSUE THREAD

**Issue by askz** [scaleway/scaleway-cli#508]
```
> scw attach xxxx                                                                                                                                                                                    [2]
You are connected, type 'Ctrl+q' to quit.
panic: runtime error: index out of range

goroutine 26 [running]:
github.com/scaleway/scaleway-cli/vendor/github.com/moby/moby/pkg/term.(*escapeProxy).Read(0xc4204cc6f0, 0xc4203c3b00, 0x80, 0x80, 0xc42000e010, 0x0, 0x0)
	/tmp/makepkg/scaleway-cli/src/src/github.com/scaleway/scaleway-cli/vendor/github.com/moby/moby/pkg/term/proxy.go:54 +0x5f5
github.com/scaleway/scaleway-cli/vendor/github.com/moul/gotty-client.(*Client).writeLoop(0xc42011c720, 0xc4202d1cf0, 0x0)
	/tmp/makepkg/scaleway-cli/src/src/github.com/scaleway/scaleway-cli/vendor/github.com/moul/gotty-client/gotty-client.go:352 +0x398
created by github.com/scaleway/scaleway-cli/vendor/github.com/moul/gotty-client.(*Client).Loop
	/tmp/makepkg/scaleway-cli/src/src/github.com/scaleway/scaleway-cli/vendor/github.com/moul/gotty-client/gotty-client.go:238 +0x110
```

**Comment by chanchiem** [comment]
I'm getting the exact same issue. I had just installed scw for the first time today. Is this something that had just happened?

**Comment by askz** [comment]
I think it happened recently yes.
Was attaching node last week

**Comment by askz** [comment]
Any update no this ? 
This is really annoying to attach in crappy web interface...

**Comment by askz** [comment]
Okay its official now.

> NO ONE CARES @ SCALEWAY

**Comment by brmzkw** [comment]
Hello,

I'm sorry it took us so long to answer. To be completely honest, we no longer have someone working full time on scaleway-cli, hence why it takes us too long to answer issues :/

Can you confirm the issue is only on OSX?

**Comment by askz** [comment]
I understand, thanks for the so awaited reply! :)

I cannot confirm for OSX, since I'm running on linux. So only Linux AFAIK

**Comment by brmzkw** [comment]
I also get the issue on OSX, but not with docker for mac. I'll try find the issue and keep you updated.

**Comment by brmzkw** [comment]
Good news, the issue has actually been fixed by 1cd3d0b5f677

I'll make a release in the following days :-)

Thanks git-bisect!
